homeassistant/components/hvv_departures/sensor.py

```diff
@@ -0,0 +1,219 @@
+"""Sensor platform for hvv."""
+from datetime import timedelta
+import logging
+
+from aiohttp import ClientConnectorError
+from pygti.exceptions import InvalidAuth
+from pygti.gti import GTI, Auth
+
+from homeassistant.const import (
+    ATTR_ATTRIBUTION,
+    ATTR_ID,
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_USERNAME,
+    DEVICE_CLASS_TIMESTAMP,
+)
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+from homeassistant.util.dt import utcnow
+
+from .const import ATTRIBUTION, DOMAIN, MANUFACTURER
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=1)
+MAX_LIST = 20
+MAX_TIME_OFFSET = 360
+ICON = "mdi:bus"
+UNIT_OF_MEASUREMENT = "min"
+
+ATTR_DEPARTURE = "departure"
+ATTR_LINE = "line"
+ATTR_ORIGIN = "origin"
+ATTR_DIRECTION = "direction"
+ATTR_TYPE = "type"
+ATTR_DELAY = "delay"
+ATTR_NEXT = "next"
+
+PARALLEL_UPDATES = 0
```

Comment:
This means that there can be unlimited parallel updates.

Reply:
Yeah i guess that's what I want? I could just make up a number but I really don't know how to choose this number

Reply:
If that's what we want, it's good. :+1: